### PR TITLE
Fix/ci no tests full genesis test

### PR DESCRIPTION
**Pull request by jferrant**
Closes https://github.com/stacks-network/stacks-core/issues/5554

### CODE REVIEW

**Review by jcnelson — APPROVED** [review]

**Review by hstove — CHANGES_REQUESTED** [review]

File: .github/workflows/stacks-core-tests.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
       max-parallel: 2
       matrix:
         test-name:
-          - neon_integrations::bitcoind_integration_test
+          - tests::neon_integrations::bitcoind_integration_test
```
[the remark this answers is not in the document]
Comment (wileyj) [reply .github/workflows/stacks-core-tests.yml:32]: in fact, i'd say we remove that entire step in the workflow - it's been failing recently anyways

[the remark this answers is not in the document]
Comment (wileyj) [reply .github/workflows/stacks-core-tests.yml]: @jferrant  - i've removed it and pushed

[the remark this answers is not in the document]
Comment (jferrant) [reply .github/workflows/stacks-core-tests.yml]: Yes I noticed this, but I assumed it was kept separately due to the test coverage call after it. @wileyj if this is redundant, let me know.

[the remark this answers is not in the document]
Comment (jferrant) [reply .github/workflows/stacks-core-tests.yml]: Oh I see you have already dealt with @wileyj  :P

**Review by obycode — DISMISSED** [review]
LGTM! Tests are failing? Just delete the tests -- problem solved! 😅

**Comment by obycode** [comment]
Are the jobs not running because of something wrong with this change?

**Comment by obycode** [comment]
> Are the jobs not running because of something wrong with this change?

I just pushed a change that I think resolves that. 🤞

**Review by obycode — APPROVED** [review]

**Review by hstove — APPROVED** [review]

**Pull request merged**